orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -110,29 +131,12 @@
     private String mCurrentStatus = STATUS_OFF;
     private TorControlConnection conn = null;
     private int mLastProcessId = -1;
-    private ArrayList<String> configBuffer = null;
-    private ArrayList<String> resetBuffer = null;
     private File fileControlPort, filePid;
     private NotificationManager mNotificationManager = null;
     private NotificationCompat.Builder mNotifyBuilder;
     private boolean mNotificationShowing = false;
-    private final ExecutorService mExecutor = Executors.newCachedThreadPool();
-    private File mHSBasePath;
+    private File mHSBasePath, mV3OnionBasePath;
     private ArrayList<Bridge> alBridges = null;
-    private final String[] hsProjection = new String[]{
-            HiddenService._ID,
-            HiddenService.NAME,
-            HiddenService.DOMAIN,
-            HiddenService.PORT,
-            HiddenService.AUTH_COOKIE,
-            HiddenService.AUTH_COOKIE_VALUE,
-            HiddenService.ONION_PORT,
-            HiddenService.ENABLED};
-    private final String[] cookieProjection = new String[]{
-            ClientCookie._ID,
-            ClientCookie.DOMAIN,
-            ClientCookie.AUTH_COOKIE_VALUE,
-            ClientCookie.ENABLED};
```

Comment:
I appreciate the code style cleanups like this.  One thing I find very helpful when reviewing code is separating out the code format and whitespace changes into their own commit.  Then it is easy to read the commits that are actually changing functionality.  Or even better, in a dedicated pull request.  One thing I frequently do when I find a bunch of code style things that need changing is to write a bunch of `// TODOs` with each code style change, then do them all in a separate pull request.